armi/utils/plotting.py

```diff
@@ -12,6 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+This module makes heavy use of matplotlib. Beware that plots generated with matplotlib
```

Comment:
You can do something like this, which will render as a big warning box on the docs:

```
 .. warning:: 
    This is a warning message
```

See: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Admonitions.html#warning

Reply:
Maybe `.. caution::` could be good instead